build/build-config.yml

```
@@ -280,6 +280,12 @@ config:
         dockerfile: "build/maven/Dockerfile"
       - work-dir: "utilities/egov-survey-services/src/main/resources/db"
         image-name: "egov-survey-services-db"
+  
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)